common/main/en.xml

```diff
@@ -9338,9 +9340,6 @@ annotations.
 	<personNames>
 		<nameOrderLocales order="givenFirst">und en</nameOrderLocales>
 		<nameOrderLocales order="surnameFirst">ja ko vi yue zh</nameOrderLocales>
-		<parameterDefault parameter="length">medium</parameterDefault>
-		<parameterDefault parameter="formality">informal</parameterDefault>		
```

Comment:
Why is this PR deleting the personName formality defaults?

Reply:
not sure... unintended, so I'll fix